### DISCUSSION TITLE
Fetching partial entities syntax on v6

### DISCUSSION THREAD

**Discussion by dungkaka** [mikro-orm/mikro-orm#5127] [General]
In v5, i can use nested object syntax for "fields", like this:
<img width="1029" alt="image" src="https://github.com/mikro-orm/mikro-orm/assets/38502025/44431612-ddd1-4a89-a12a-cdd806799ca1">
But, in v6, it show error syntax:
<img width="635" alt="image" src="https://github.com/mikro-orm/mikro-orm/assets/38502025/5c082084-bac5-4af9-8310-4510a8413112">

Are there some change about this ?

**Comment by B4nan** [comment]
Yes, this alternative object syntax is no longer supported, not sure where you came up with the screenshot, I guess in v5 docs?

**Reply by dungkaka** [reply]
Yes, It is in v5 docs, I compare document of v5 and v6 and I realize that this syntax not exist in v6, and I also not see in upgrade guide from v5 to v6, then I open a discussion.

**Reply by B4nan** [reply]
It was removed with the strict partial loading support, I think there is some mention on this in the upgrading guide but its not very explicit. 

Technically it still works but not on type level.
